lib/datadog/open_feature/exposures/event.rb

```diff
@@ -0,0 +1,82 @@
+# frozen_string_literal: true
+
+require_relative '../../core/utils/time'
+
+module Datadog
+  module OpenFeature
+    module Exposures
+      # A data model for an exposure event
+      class Event
+        TARGETING_KEY_FIELD = 'targeting_key'
+        ALLOWED_FIELD_TYPES = [
+          String,
+          Integer,
+          Float,
+          TrueClass,
+          FalseClass
+        ].freeze
+
+        class << self
+          def build(result, flag_key:, context:)
+            payload = {
+              timestamp: current_timestamp_ms,
+              allocation: {
+                key: result.allocation_key
+              },
+              flag: {
+                key: flag_key
+              },
+              variant: {
+                key: result.variant
+              },
+              subject: {
+                id: context.targeting_key,
+                attributes: extract_attributes(context)
+              }
+            }
+
+            new(payload)
+          end
+
+          private
+
+          def extract_attributes(context)
```

Comment:
Can you add a short method-level comment to document what this method does (succinct is totally fine).